spec/Section 4 -- Introspection.md

```diff
@@ -359,14 +382,14 @@ each item in the list.
 The modified type in the `ofType` field may itself be a modified type, allowing
 the representation of Lists of Lists, or Lists of Non-Nulls.
 
-Fields
+Fields\:
 
 * `kind` must return `__TypeKind.LIST`.
-* `ofType`: Any type.
+* `ofType` may be any type.
```

Comment:
`may` here sounds right in English, but in must/should/may parlance it makes it sound like `ofType` is optional which it is not. (Also I've added `other` to make it clear it cannot return itself... but feel free to trim that back out.)

```suggestion
* `ofType` must return any other type.
```

Reply:
I think "any other type" could be misread as List of Lists not being valid

Reply:
Yeah I considered that, but figured List isn’t itself a type; it has to be a list _of_ something. But I agree 👍